beacon_node/lighthouse_network/src/rpc/protocol.rs

```diff
@@ -542,8 +574,13 @@ impl ProtocolId {
                 <LightClientBootstrapRequest as Encode>::ssz_fixed_len(),
                 <LightClientBootstrapRequest as Encode>::ssz_fixed_len(),
             ),
+            // TODO(lc-update) make sure rpc limits are set correctly for lc data
```

Comment:
Both `LightClientOptimisticUpdate` and `LightClientFinalityUpdate` have empty requests, so the limits look ok. And `LightClientUpdatesByRangeRequest` has fixed size and your min / max functions are correct.